h-include.js

```diff
@@ -42,7 +42,7 @@ var hinclude;
     classprefix: "include_",
 
     show_content: function (element, req) {
-      var i, include, message, fragment = element.getAttribute('fragment');
+      var i, include, message, fragment = element.getAttribute('fragment') || 'body';
       if (req.status === 200 || req.status === 304) {
         var container = document.implementation.createHTMLDocument().documentElement;
         container.innerHTML = req.responseText;
```

Comment:
And this throws in IE9 because the `<html>` element doesn't like `.innerHTML`. 
And using 

``` .html
document.implementation.createHTMLDocument(' ').body = req.responseText;
```
 would leave `<head>` elements in the content. 
I don't currently know of a simple way to support HTML document parsing on IE9.